sandbox/modules/identity/src/main/java/org/opensearch/identity/rest/user/put/PutUserAction.java

```diff
@@ -6,21 +6,21 @@
  * compatible open source license.
  */
 
-package org.opensearch.identity.rest.user.create;
+package org.opensearch.identity.rest.user.put;
```

Comment:
I would expect this class to be update*, why use put?

Reply:
We handle PUT loosely, meaning PUT can be used to create and update in our case. And since `create` or `update` sound one-directional, i decided to rename it to `put`. Follow PR's will have `get` instead of `read` package and 
 a `delete` package